email-notification-service/README.md

```diff
@@ -27,51 +27,7 @@ npm install <Driver> --save
-| EMAIL_SUBJECT               | Trubudget Notification            | This is injected into the `subject` field of the email notification                                                                                                                                                                                    |
-| EMAIL_TEXT                  | You have received a notification. | This is injected into the `body` of the email notification                                                                                                                                                                                             |
-| LOG_LEVEL                   | info                              | Defines the log output. Supported levels are `trace`, `debug`, `info`, `warn`, `error`, `fatal`                                                                                                                                                        |
-| PRETTY_PRINT                | false                             | Decides whether the logs printed by the email service are pretty printed or not.                                                                                                                                                                       |
-| JWT_SECRET                  | - (required)                      | A secret of min length of 32 - It is used to verify the JWT_TOKEN sent by users of the email-service endpoints                                                                                                                                         |
-| AUTHENTICATION              | JWT                               | If set to none, no JWT-Token is required for all endpoints. If set JWT, a JWT token is necessary                                                                                                                                                       |
-| ACCESS_CONTROL_ALLOW_ORIGIN | "\*"                              | Since the service uses CORS, the domain by which it can be called needs to be set. Setting this value to `"*"` means that it can be called from any domain. Read more about this topic [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). |
-
-### Blockchain
-
-For details see [Blockchain environment variables](../blockchain/README.md#email-service).
-
-### Frontend
-
-For details see [Blockchain environment variables](../frontend/README.md#email-service).
-
-#### JWT_SECRET
-
-The JWT_SECRET is shared between Trubudget's blockchain api and email-service. The endpoints of the email-service can
-only be used by providing a valid JWT_TOKEN signed with this JWT_SECRET. Since the blockchain is using the notification
-endpoints and the ui is using the user endpoints the secret has to be shared.
+See [Environment Variables](./environment-variables.md)
```

Comment:
General comments about this README file:

- [ ]  there's a block that's duplicated ([Getting Started](https://github.com/openkfw/TruBudget/blob/c3a0c7daefa3d12ed73b78880340d400352ef57e/email-notification-service/README.md#getting-started) and [Enable email notifications](https://github.com/openkfw/TruBudget/blob/c3a0c7daefa3d12ed73b78880340d400352ef57e/email-notification-service/README.md#enable-email-notifications).
- [ ] the section mentioned above mentions a pre-set docker-compose cluster which starts the whole TruBudget application including all email components - we don't have that anymore 
- [ ] startDev.sh script has to be removed, as it uses an inexistent docker-compose
- [ ] [`Endpoints`](https://github.com/openkfw/TruBudget/blob/c3a0c7daefa3d12ed73b78880340d400352ef57e/email-notification-service/README.md#endpoints) is a subsection of `Disable email notifications` - I think this is a mistake